src/main/java/seedu/todo/controllers/ClearController.java

```diff
@@ -18,16 +18,35 @@
 import seedu.todo.controllers.concerns.Tokenizer;
 import seedu.todo.models.TodoListDB;
 
+/**
+ * Controller to clear CalendarItems.
+ * 
+ * @author Tiong Yaocong
+ *
+ */
 public class ClearController implements Controller {
     
     private static final String NAME = "Clear";
     private static final String DESCRIPTION = "Clear all tasks/events or by specify date.";
-    private static final String COMMAND_SYNTAX = "clear [task/event] on [date]";
+    private static final String COMMAND_SYNTAX = "clear [task/event] [on date]";
     private static final String COMMAND_WORD = "clear";
     private static final String MESSAGE_CLEAR_NO_ITEM_FOUND = "No item found!";
     private static final String MESSAGE_CLEAR_SUCCESS = "A total of %s deleted!\n" + "To undo, type \"undo\".";;
-    private String invalidDate = null;
-    
+    private static final String INVALID_DATE = null;
+    private static final String NOT_FOUND = null;
+    private static final String[] PARSED_RESULT_NOT_FOUND = null;
+    private static final LocalDateTime DATE_NOT_FOUND = null;
+    //Use by array access
+    private static final int KEYWORD = 0;
+    private static final int RESULT = 1;
+    private static final int MAXIMUM_SIZE = 2;
+    //Use by accessing date value
+    private static final int DATE_ON = 0;
+    private static final int DATE_FROM = 1;
+    private static final int DATE_TO = 2;
```

Comment:
These constants should be prefixed with `INDEX_`, as per [style guide](https://oss-generic.github.io/process/codingstandards/coding-standards-java.html), point 13.